### ISSUE TITLE
Decide which page is the main, delete duplicated description of the project

### ISSUE THREAD

**Issue by ghost** [jilltxt/machinevision#144]
~~It's a bit confusing about which page is the main one. The link on the project website leads here: “https://machine-vision.no/knowledgebase”. But there is also a page at https://machine-vision.no/ with another, repetitive description of a project. The latter one is placed under the title “Technologies” in the menu. It is counterintuitive and confusing.~~

I see two options here
1) Set https://machine-vision.no/knowledgebase as the main page
- change link from https://machine-vision.no/ to https://machine-vision.no/technologies/
- delete description of the project from "Technologies"

2) Set "Technologies" as the main page
- delete the "technologies" from the menu
- change the link to the database on the UiB website to https://machine-vision.no/
- delete description of the project from https://machine-vision.no/knowledgebase

**Comment by steinmb** [comment]
In lack of polished front page we decided to use the landing page `/technologies` as the front page. This was decided after the menu item was created and we kept it around as a extra navigation point allowing users to navigate to this page when viewing, example an article. Perfect it is not. As you mention. Navigating to `technologies` viewing the front page is confusing, nothing really happen.

We still need designers to come up with a project front page if we are not keeping the one we currently have. I personally do not think https://machine-vision.no/knowledgebase is a fitting front page. Text heavy and long.

Old and replaced front page is found at: https://machine-vision.no/front-page
